# CPG Coordinate System and Initial Conditions.
lat0:  0.0 #35.5738022685142 
lon0: 0.0 # 129.19160629798
yaw0: 0.0
X0:   0.0
Y0:   0.0
Psi0:  0.0  #-0.149
V0:   0.0
